src/modules/components/proposal/proposalActions/proposalActionsItem/proposalActionsItem.tsx

```diff
@@ -30,6 +31,10 @@ export const ProposalActionsItem = <TAction extends IProposalAction = IProposalA
     const { copy } = useGukModulesContext();
     const { buildEntityUrl } = useBlockExplorer({ chainId });
 
+    const chains = useChains();
+    const chain = chains.find((chain) => chain.id === chainId);
```

Comment:
I think if I remember correctly, @cgero-eth recommended adding the symbol to the network definitions so that we can easily reuse this elsewhere when needed.